packages/broker/test/integration/plugins/storage/dataMetadataEndpoint.test.ts

```diff
@@ -72,14 +74,8 @@ describe('dataMetadataEndpoint', () => {
         expect(res.lastMessage).toEqual(0)
     })
 
-    async function setUpStream(): Promise<Stream> {
-        const freshStream = await createTestStream(client1, module)
-        await freshStream.addToStorageNode(storageNodeAccount.address)
-        return freshStream
-    }
-
     it('returns (non-zero) metadata for existing stream', async () => {
-        const stream = await setUpStream()
```

Comment:
Is it necessary to do the stream creation in `beforeAll`? In the previous implementation the stream was created for only the one test case which uses it.